### ISSUE TITLE
Caribu does not allow setting reflectance = transmitance

### ISSUE THREAD

**Issue by christian34** [openalea/caribu#15]
When setting reflectance = transmitance, -1 is returned as Ei, with error messages in the log

**Comment by mchelle** [comment]
Yes; it is normal because Ei from teh 2 faces of a triangle is calculated from the radiosity Bi by solving a system of 2 lienar equations wirh 2 unknowns. and when refl = trans, the two equations are colinear, thus the system can not be solved. 
Michaël Chelle
